README.md

```diff
@@ -0,0 +1,186 @@
+version of the package being registered is compatible with every possible future
+version of Julia after (and including) `1.3`, but that is impossible to know.
+Instead,
+
+```toml
+[compat]
+julia = "1.3"
+```
+
+declares compatibility with Julia `1.3`, `1.4`, etc., including any release after
+`1.3` but strictly before `2.0`, since none of those releases should break code
+relying on the public API of Julia 1.3. (To specify only `1.3` and its patch
+releases, use `julia= "~1.3"`.)
+
+To claim compatibility with
+more than one breaking release of a package, add multiple entries such as
+
+```toml
+[compat]
+StaticArrays = "0.10, 0.11"
+```
+
+This attests that the package is compatible with both versions of `StaticArrays`
+even though, according to semantic versioning (which is discussed in the `Pkg`
+documentation
+[here](https://julialang.github.io/Pkg.jl/dev/compatibility/#Version-specifier-format-1)),
+the `0.11` release could have broken code relying on the `0.10` release's public
+API.
+
+See [the `Pkg`
+documentation](https://julialang.github.io/Pkg.jl/v1/compatibility/) for more on
+how to specify `compat` entries.
+
+Note that standard libraries are currently coupled to the Julia version, and
+don't have separate version numbers. So while they are declared as dependencies,
+they don't need separate entries in the `compat` section from Julia itself.
+
+### Merge conflicts
+
+If the pull request against the General registry has merge conflicts with the
+General registry itself, the pull request cannot be merged. In this case, just
+retrigger the Registrator.jl bot (either add another comment with
+"@JuliaRegistrator register", or use the web interface), and it will update the
+pull request.
+
+### Julia Package naming conventions
+
+The guidelines for naming packages are listed
+[here](https://julialang.github.io/Pkg.jl/v1/creating-packages/#Package-naming-guidelines-1)
+in the package manager's documentation. If a new package's name does not adhere
+to those guidelines or could otherwise cause confusion, maintainers of the
+General registry or the Julia community might comment on the pull request to ask
+that the name be changed, or why the usual conventions were not used in this
+case. Additionally, the automerge bot will not merge pull requests corresponding
+to new packages whose names do not meet certain requirements. The package can
+still be added by manually merging the request, however, and updates to the
+package will still be automerged as usual.
+
+If you think that your package should not follow those conventions for some
+reason or another, just explain why. Otherwise, it is often a good idea to just
+rename the package-- it is more disruptive to do so after it is already
+registered, and sticking to the conventions makes it easier for users to
+navigate Julia's many varied packages.
+
+As long as the package is not yet registered, renaming the package from
+`OldName.jl` to `NewName`.jl is reasonably straightforward:
+
+
```

Comment:
There's no need for two blank lines here, right?

Reply:
True! Fixed